python/llm/src/bigdl/llm/ggml/model/bloom/bloom.py

```diff
@@ -87,15 +109,73 @@ def __init__(self,
         self.ctx = bloom_load(bytes(model_path, encoding='utf-8'), n_ctx, n_threads)
         invalidInputError(self.ctx is not None, f"Failed to load model from {model_path}")
         self.n_ctx = n_ctx
+        self.n_parts = n_parts
+        self.n_gpu_layers = n_gpu_layers
+        self.f16_kv = f16_kv
         self.seed = seed
```

Comment:
I compared the default values of the parameters in langchain integrations. https://github.com/intel-analytics/BigDL/blob/main/python/llm/src/bigdl/llm/langchain/llms/bigdlllm.py. Most of the values are the same as the default values here. 

However there're some differences:
- `seed = -1` is -1 working in bloom?  https://github.com/intel-analytics/BigDL/blob/main/python/llm/src/bigdl/llm/langchain/llms/bigdlllm.py#L98
- `n_gpu_layers = None` https://github.com/intel-analytics/BigDL/blob/main/python/llm/src/bigdl/llm/langchain/llms/bigdlllm.py#L120

Please kindly check which is the proper default values. Maybe

Reply:
> * `seed = -1` is -1 working in bloom?  https://github.com/intel-analytics/BigDL/blob/main/python/llm/src/bigdl/llm/langchain/llms/bigdlllm.py#L98

if `seed < 0`, bloom will use current timestamp as random seed

Reply:
LGTM. I'll validate langchain bloom path after you merge this PR.